src/web-animations-next-animation.js

```diff
@@ -224,6 +224,21 @@
         this._onfinish = this._animation.onfinish;
       }
     },
+    get oncancel() {
+      return this._oncancel;
+    },
+    set oncancel(v) {
+      if (typeof v == 'function') {
+        this._oncancel = v;
+        this._animation.oncancel = (function(e) {
```

Comment:
This could be a little simpler, we don't need the `this._oncancel` property, we can store directly in `this._animation.oncancel`.